### PR TITLE
added redeemed text, compressed images

### PR DESCRIPTION
**Pull request by developerpeachy**
IMG_2667

- Added redeemed text for the ticket
- Compressed the background image and logo. 
- Background image: 2.98 KB
- Logo: 290.80 KB 
- used https://kraken.io/web-interface to compress but let me know if you got any ideas on image compression :)

### CODE REVIEW

**Comment by developerpeachy** [comment]
oh also made the corners curved a bit more.

File: examples/UEFA/UEFA.en.shtml [hunk 1]
```diff
@@ -113,6 +113,8 @@ class Token {
             redeemedMessage = "(Redeemed)"
         } else if(this.props.redeemed === "0") {
             redeemedMessage = "(Not redeemed)"
+        } else {
+            redeemedMessage = "Redeem Error"
```
Comment (James-Sangalli) [inline examples/UEFA/UEFA.en.shtml]: @developerpeachy can we have it that it simply displays nothing if not redeemed or there is an error?

**Comment by hboon** [comment]
> * Logo: 290.80 KB
> * used https://kraken.io/web-interface to compress but let me know if you got any ideas on image compression :)

@developerpeachy Have you tried [ImageOptim](https://imageoptim.com) for PNG and [ShrinkIt](https://panic.com/blog/shrinkit-1-2/) for PDF?

**Comment by developerpeachy** [comment]
@hboon no, I can give this a go. The file types i have are .png and .svg though.

**Comment by James-Sangalli** [comment]
Closes https://github.com/AlphaWallet/TokenScript/issues/290

**Pull request merged**